src/paperoni/config.py

```diff
@@ -28,10 +28,40 @@ class PaperoniConfig:
     api_keys: Keys[str, str] = field(default_factory=Keys)
     fetch: TaggedSubclass[Fetcher] = field(default_factory=RequestsFetcher)
     focuses: Focuses = field(default_factory=Focuses)
+    autofocus: AutoFocus[str, AutoFocus.Type] = field(default_factory=AutoFocus)
     refine: Refine = None
     work_file: Path = None
     collection: TaggedSubclass[PaperCollection] = None
 
+    def __post_init__(self):
+        # Only used for type hinting
+        self._file: Path = getattr(self, "_file", None)
+        # Ideally we would not need this but there seams to an issue with
+        # autofocus not being of type AutoFocus when using something like:
```

Comment:
Yeah, turns out it's the same problem for Keys actually. The next version of serieux will fix that.